Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -3317,3 +3317,43 @@
 
 /* Info alert when editing a generated column */
 "The current field \"%@\" is a generated column and therefore cannot be edited." = "The current field \"%@\" is a generated column and therefore cannot be edited.";
+
+/* Pin a view to the top of the list of tables
+ Table List : Context Menu : Pin View
+ Table List : Gear Menu : Pin View */
```

Comment:
Maybe worth making this comment a single line and a little more direct like the other localization string descriptions are
ex. `/* Menu option to pin a view to the top of the list of tables */`